### ISSUE TITLE
Live updating doesn't work client side with Visual Studio Code Live Share Extension

### ISSUE THREAD

**Issue by snoopen** [wallabyjs/quokka#206]
### Issue description or question

If you join a shared coding session with VS Live Share and start Quokka on a shared file, you get the correct output once but updates don't appear to get processed. The executed filename does not match the open filename.

Quokka works fine server side in a shared session. Only client side is affected.

#### vsliveshare.js
```javascript
const test = 1;
test
```

### Quokka.js Console Output

```
​​​​Quokka #1 (node: v8.10.0)​​​​
​​​​​1​​​​​ at ​​​test​​​ ​quokka.js:2:0​
```

#### Expected result
Updating `test = 2` should output `2 at test ​vsliveshare.js:2:0​`.

#### Actual result
Updating `test = 2` outputs `1 at test ​​​quokka.js:2:0​`, with no apparent refresh.

### Code editor version

Server side:
macOS Sierra
Visual Studio Code v1.23.1
VS Live Share v0.3.198

Client side:
Windows 10
Visual Studio Code v1.23.1
VS Live Share v0.3.198

### OS name and version

Windows 10 and
macOS Sierra (OSX)

**Comment by ArtemGovorov** [comment]
I'm actually quite surprised to hear that Quokka starts at all on the client side, since AFAIK there are no real (physical) files on the client side, installed local modules etc. on the client side. 

While we can investigate if we can make some simple stuff work on the client side, I'm concerned that we'll quickly hit VS Live Share limitations (as a technology) to run things that are using local files/installed modules.

**Comment by snoopen** [comment]
Quokka runs fine on the temporary files it creates, so I didn't expect there to be much difference apart from the filename displayed. Perhaps that's a gross over simplification.

Could it be that VS Live Share would need to better establish how local files are made available to extensions?

**Comment by ArtemGovorov** [comment]
You were right, making it work for a temporary file with simple (self-sufficient) code in it was not too much trouble. We have just published the new version 1.0.125 that works for simple things.

Note that any local files/modules import will not work on the client side for the reasons described above (because the files aren't locally available to node.js to run them).

**Comment by snoopen** [comment]
Amazing, love it!

I imagine by the time you get to imports it'd be preferable to use the shared debugging in VS Live Share rather than trying to get Quokka on it. Though the client can request any file in the workspace I can imagine if even possible the cost to retrieve all the imports would be excessive.
